sdk/identity/identity/review/identity.api.md

```diff
@@ -210,6 +212,63 @@ export class ManagedIdentityCredential implements TokenCredential {
     getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken>;
     }
 
+// @public
+export enum RegionalAuthority {
+    AsiaEastValue = "eastasia",
+    AsiaSouthEastValue = "southeastasia",
+    AustraliaCentral2Value = "australiacentral2",
+    AustraliaCentralValue = "australiacentral",
+    AustraliaEastValue = "australiaeast",
+    AustraliaSouthEastValue = "australiasoutheast",
+    AutoDiscoverRegion = "AUTO_DISCOVER",
```

Comment:
It's interesting that MSAL for node seems to use a different constant for this than .NET, which uses the string constant "TryAutoDetect" which it exposes as a constant `AttemptRegionDiscovery`. Does MSAL for node have a constant defined for this? If so I think we should use it to ensure we're always in sync with this value.

Reply:
I don’t see it here: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/regional-authorities.md

I’ll ask the MSAL team.

What is the value of the `AutoDiscoverRegion` in .Net? I don’t quite understand it in the .NET’s PR. What would be the equivalent for JS on the public API perspective? I can hide the internals.

Reply:
As discussed, I’ll use the value `AutoDiscoverRegion` for now.